### ISSUE TITLE
Support for transparent video

### ISSUE THREAD

**Issue by woniu163** [linkedin/LiTr#180]
support for transparent video by libvpx, like webm

**Comment by izzytwosheds** [comment]
Technically we should already support this. VPx support is available on many devices via MediaCodec API which LiTr uses. The trick is to figure out correct `MediaMuxerMediaTarget` and `MediaCodecDecoder` parameters. I will try to take a quick look at this.

**Comment by izzytwosheds** [comment]
Can you describe your use case? Are you trying to transcode a WEBM with transparent video into MP4 with some effects? Or are you trying to create a WEBM with transparency?

**Comment by woniu163** [comment]
Yes, we tried to combine a webm and a mp4 video into a new mp4 file, but it didn't work

**Comment by izzytwosheds** [comment]
Did you try to draw webm frames onto mp4 frames? And alpha channel didn't work, I presume?

**Comment by woniu163** [comment]
yes，the alpha channel didn't work
